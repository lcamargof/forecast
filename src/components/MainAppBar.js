import React from 'react';
import { AppBar, Toolbar, Typography } from 'material-ui';

const MainAppBar = () => (
  <AppBar position="static" color="primary">
    <Toolbar>
      <Typography variant="title" color="inherit">
        Your Forecast
      </Typography>
    </Toolbar>
  </AppBar>
);

export default MainAppBar;