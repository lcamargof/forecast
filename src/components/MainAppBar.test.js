import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import MainAppBar from './MainAppBar';

describe('Component: MainAppBar', () => {
  it('Should render successfully and match the snapshot', () => {
    const component = shallow(<MainAppBar />);
    expect(shallowToJson(component)).toMatchSnapshot();
  });
});