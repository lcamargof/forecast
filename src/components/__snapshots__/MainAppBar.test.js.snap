// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: MainAppBar Should render successfully and match the snapshot 1`] = `
<WithStyles(AppBar)
  color="default"
  position="static"
>
  <WithStyles(Toolbar)>
    <WithStyles(Typography)
      color="inherit"
      variant="title"
    >
      Your Forecast
    </WithStyles(Typography)>
  </WithStyles(Toolbar)>
</WithStyles(AppBar)>
`;
